fuel-asm/src/args.rs

```diff
@@ -50,218 +50,221 @@ crate::enum_try_from! {
         /// Set `$rA` to `tx.type`
         Type = 0x001,
 
-        /// Set `$rA` to `tx.gasPrice`
-        ScriptGasPrice = 0x002,
-
         /// Set `$rA` to `tx.gasLimit`
-        ScriptGasLimit = 0x003,
-
-        /// Set `$rA` to `tx.maturity`
-        ScriptMaturity = 0x004,
+        ScriptGasLimit = 0x002,
 
         /// Set `$rA` to `tx.scriptLength`
-        ScriptLength = 0x005,
+        ScriptLength = 0x003,
 
         /// Set `$rA` to `tx.scriptDataLength`
-        ScriptDataLength = 0x006,
+        ScriptDataLength = 0x004,
 
         /// Set `$rA` to `tx.inputsCount`
-        ScriptInputsCount = 0x007,
+        ScriptInputsCount = 0x005,
 
         /// Set `$rA` to `tx.outputsCount`
-        ScriptOutputsCount = 0x008,
+        ScriptOutputsCount = 0x006,
 
         /// Set `$rA` to `tx.witnessesCount`
-        ScriptWitnessesCound = 0x009,
+        ScriptWitnessesCount = 0x007,
 
         /// Set `$rA` to `Memory address of tx.receiptsRoot`
-        ScriptReceiptsRoot = 0x00A,
+        ScriptReceiptsRoot = 0x008,
 
         /// Set `$rA` to `Memory address of tx.script`
-        Script = 0x00B,
+        Script = 0x009,
 
         /// Set `$rA` to `Memory address of tx.scriptData`
-        ScriptData = 0x00C,
+        ScriptData = 0x00A,
 
         /// Set `$rA` to `Memory address of tx.inputs[$rB]`
-        ScriptInputAtIndex = 0x00D,
+        ScriptInputAtIndex = 0x00B,
 
         /// Set `$rA` to `Memory address of t.outputs[$rB]`
-        ScriptOutputAtIndex = 0x00E,
+        ScriptOutputAtIndex = 0x00C,
 
         /// Set `$rA` to `Memory address of tx.witnesses[$rB]`
-        ScriptWitnessAtIndex = 0x00F,
-
-        /// Set `$rA` to `tx.gasPrice`
-        CreateGasPrice = 0x010,
-
-        /// Set `$rA` to `tx.gasLimit`
-        CreateGasLimit = 0x011,
-
-        /// Set `$rA` to `tx.maturity`
-        CreateMaturity = 0x012,
+        ScriptWitnessAtIndex = 0x00D,
```

Comment:
Why does it skip from `0x00D` to `0x100`?

Reply:
It looks like jumps on a few of them.

Reply:
The values are from the [specification](https://github.com/FuelLabs/fuel-specs/pull/514/files#diff-ee3e0bbd57928f651eca1142f65942e1592743ec5fd20e20b8a44c9998679b06)